packages/edge-config/src/utils/io-boundary.ts

```diff
@@ -0,0 +1,50 @@
+let blockedBoundary: null | Promise<void> = null;
+
```

Comment:
_This comment is in random place so we can have a threaded discussion_

> While we could argue that the kind of data edge-config provides is prerenderable because it has a long lifetime it is not clear that users expect this.

I'm fully on board that the default expectation is for this to be IO and not part of the prerender. For, [we have this explicit `no-store` configuration option](https://github.com/vercel/storage/blob/main/packages/edge-config/src/index.ts#L278), that is used by some users of this library to treat Edge Config more like a CMS. Would that still work with the proposed changes?